python/equipage/tests.py

```diff
@@ -235,6 +235,16 @@ def test_qpid_proton_cpp_subscriptions(session):
             call("build/send {0} t1 abc", server.connection_url)
             call("build/subscriptions/durable-shared-subscribe {0} t1 1", server.connection_url)
 
+def test_qpid_proton_cpp_link_failure_recovery(session):
+    with working_dir(join(session.examples_dir, "qpid-proton-cpp")):
+        with TestServer() as server:
+            # Test where receiver closes link, sender must handle error
+            with start_process("build/reconnect/direct-rcv-close-link"):
```

Comment:
What does direct mean here?  That's not the terminology the Equipage examples use.  I think you mean it's acting as a server.

Also, Equipage doesn't have "rcv" in example names.  It's spelled out.